drop-boxes/register-omero-metadata/image_registration_process.py

```diff
@@ -109,6 +144,27 @@ def extractMetadataFromTSV(self, tsvFilePath):
     def registerExperimentDataInOpenBIS(self):
         pass
 
+    def registerKeyValuePairs(self, image_id, property_map):
+        cmd_list = list(self._init_cmd_list)
+
+        #string format: key1::value1//key2::value2//key3::value3//...
+        key_value_str = ""
+        for key in property_map.keys(): 
+            key_value_str = key_value_str + str(key) + "::" + str(property_map[key]) + "//"
+        key_value_str = key_value_str[:len(key_value_str)-2] #remove last two chars
+
+        cmd_list.append( "python backendinterface.py -i " + str(image_id) + " -a " + key_value_str )
+
+        commands = ""
+        for cmd in cmd_list:
+            commands = commands + cmd + "\n"
+
+        process = Popen( "/bin/bash", shell=False, universal_newlines=True, stdin=PIPE, stdout=PIPE, stderr=PIPE )
+        out, err = process.communicate( commands )
+
+
+        return 0
```

Comment:
Key value pair is very technical, can you make it more expressive to follow the process please?
Like:
```suggestion
    def registerUnstructuredMetadata(self, image_id, property_map):
        cmd_list = list(self._init_cmd_list)

        #string format: key1::value1//key2::value2//key3::value3//...
        key_value_str = ""
        for key in property_map.keys(): 
            key_value_str = key_value_str + str(key) + "::" + str(property_map[key]) + "//"
        key_value_str = key_value_str[:len(key_value_str)-2] #remove last two chars

        cmd_list.append( "python backendinterface.py -i " + str(image_id) + " -a " + key_value_str )

        commands = ""
        for cmd in cmd_list:
            commands = commands + cmd + "\n"

        process = Popen( "/bin/bash", shell=False, universal_newlines=True, stdin=PIPE, stdout=PIPE, stderr=PIPE )
        out, err = process.communicate( commands )


        return 0
```

Reply:
I would leave it as it is, since "key-value pairs" is the standard nomenclature of OMERO, and it refers directly to the type of annotation used in the omero server

Reply:
ok, would you mind and add a method docstring and maybe add `omero` in the method name to make it very explicit.

Reply:
done